doc/api/Renter.md

```diff
@@ -269,8 +281,15 @@ uploads a file to the network from the local filesystem.
 
 ###### Query String Parameters
 ```
+// The number of data pieces to use when erasure coding the file.
+datapieces // int
+
+// The number of parity pieces to use when erasure coding the file. Total
+// redundancy of the file is (datapieces+paritypieces)/datapieces.
+paritypieces // int
```

Comment:
haha yeah, I was just happy to see that you updated the docs to reflect the API change